### ISSUE TITLE
VirtualWebappLoader does not work on Tomcat 8

### ISSUE THREAD

**Issue by btshimizukza** [terasolunaorg/terasoluna-gfw-functionaltest#124]
We need change the configuration when will support the tomcat 8 as test environment.
see [tomcat 8 migration guide](https://tomcat.apache.org/migration-8.html#Web_application_resources).

```
     <Loader className="org.apache.catalina.loader.VirtualWebappLoader"
         virtualClasspath="/home/terasoluna/gfw/webapps-env-jars/terasoluna-gfw-functionaltest-env-tomcat-postgresql/*.jar" />
     <JarScanner scanAllDirectories="true" />
```

Reference Site:
http://stackoverflow.com/questions/23143697/adding-external-resources-to-class-path-in-tomcat-8

**Comment by making** [comment]
@btkodakak Please check this issue

**Comment by kazuki43zoo** [comment]
resolved via #198 and #199 .
